Submitty/Submitty
# [BUGFIX:UI] Incorrect Late Days Banner

### Please check if the PR fulfills these requirements:


### What is the current behavior?
Closes #3534 and closes #3125 and closes #3184 
![Screenshot from 2019-07-31 12-54-10](https://user-images.githubusercontent.com/36307579/62231856-d9ab7380-b392-11e9-8ab0-839ee9321d2f.png)
![Screenshot from 2019-07-31 12-54-31](https://user-images.githubusercontent.com/36307579/62231857-d9ab7380-b392-11e9-846b-c5feb80e5305.png)
![Screenshot from 2019-07-31 12-55-28](https://user-images.githubusercontent.com/36307579/62231858-da440a00-b392-11e9-8100-9f5d8be9bc85.png)
![Screenshot from 2019-07-31 12-55-56](https://user-images.githubusercontent.com/36307579/62231859-da440a00-b392-11e9-9f47-e183fd5db062.png)
![Screenshot from 2019-07-31 12-57-34](https://user-images.githubusercontent.com/36307579/62231860-da440a00-b392-11e9-8fc2-a60043ea58cf.png)

### What is the new behavior?
If you want to resubmit late on the same day you have already submitted late, it tells you how many late days you have left and that it will cost 0 to resubmit now. The next day, it will tell you it will cost 1 to resubmit now.
![Screenshot from 2019-07-31 12-01-10](https://user-images.githubusercontent.com/36307579/62230075-0b224000-b38f-11e9-803b-008553accef2.png)
![Screenshot from 2019-07-31 12-03-49](https://user-images.githubusercontent.com/36307579/62230081-0e1d3080-b38f-11e9-9c5d-f38933253568.png)
![Screenshot from 2019-07-31 12-04-20](https://user-images.githubusercontent.com/36307579/62230084-0fe6f400-b38f-11e9-87af-022606fd882b.png)
![Screenshot from 2019-07-31 12-04-58](https://user-images.githubusercontent.com/36307579/62230086-11b0b780-b38f-11e9-8be7-56b853b06d18.png)
![Screenshot from 2019-07-31 12-23-18](https://user-images.githubusercontent.com/36307579/62230089-137a7b00-b38f-11e9-8cd5-5a8e0fa75bf1.png)

### Other information?
How to test:
As the instructor, set the homework to have a reasonable release date, make sure you give the option to have late days (click pencil, go to dates)
vagrant ssh
install submitty
disable ntp: sudo systemctl disable ntp
run command: timedatectl set-ntp 0
sudo date -s "2019-xx-xx xx:xx:xx" (fill x's with desired days/times 1 day after due date)
check if date has taken by running: date
refresh and log in to student page
submit and look at message
submit as many times as you want
sudo date -s "2019-xx-xx xx:xx:xx" (fill x's with desired days/times 2 days after due date)
refresh and note the message
submit as much as you want
when done testing, simply reenable what you disabled
enable ntp: sudo systemctl enable ntp
run command: timedatectl set-ntp 1
